src/Time.cpp

```diff
@@ -88,6 +88,32 @@ timeval Time::toTimeval() const
     return tv;
 }
 
+std::vector<int> Time::toTimeValues() const
+{
+    int64_t microseconds = this->microseconds;
+
+    int days = microseconds / 86400000000;
+    microseconds -= days * 86400000000;
```

Comment:
This line (`days * 86400000000`) will overflow for `days>10`, which is undefined behavior (i.e., it might accidentally work, but it could result in bogus results).

Reply:
Hm, thinking about it again, 86400000000 is already greater than max-int32, so this should actually work as intended (I made a `/` vs `*` error when first looking at it).